scripts/init_supabase.py

```
@@ -0,0 +1,123 @@
+"""
+Supabase database initialization script.
+
+This script creates the necessary tables and initializes sample data in Supabase.
+"""
+
+import streamlit as st
```

Ruff (0.8.2)

`streamlit` imported but unused

Remove unused import: `streamlit`

(F401)